### ISSUE TITLE
phpcs-variable-analysis 2.6.2 is missing a bunch of bug fixes

### ISSUE THREAD

**Issue by sirbrillig** [Automattic/amp-wp#4882]
## Bug Description

This package currently depends on phpcs-variable-analysis 2.6.2, but that package version is missing many bug fixes present in the current version (2.8.2 at the time of this writing).

(Relatively minor, but the impetus for this issue is that the Tests directory was included in 2.6.2 and prior to sirbrillig/phpcs-variable-analysis@9d86ccb and sirbrillig/phpcs-variable-analysis@10eea96 the fixtures were not valid PHP, which could cause errors if the vendor directory is being scanned, eg: by WP VIP automated processes.)

**Comment by schlessera** [comment]
Well, even though I agree we should keep dependencies up-to-date as best we can, I wouldn't say being on the second-to-last minor should be considered a bug...

Is there something specific that is breaking for you because of that?

**Comment by sirbrillig** [comment]
Hah. Sorry, I got a bug report on the package, but it turns out it was from on old version of _this_ package. Nevermind!

**Comment by westonruter** [comment]
@sirbrillig We have a PR pending to update in #4803. It's blocked on another PR, so that's why we haven't updated yet.
